rpm/tracker.spec

```diff
@@ -1,6 +1,6 @@
 Name:       tracker
 Summary:    Desktop-neutral metadata database and search tool
-Version:    3.2.0
+Version:    3.2.1
 Release:    1
 License:    LGPLv2+ and GPLv2+
 URL:        https://wiki.gnome.org/Projects/Tracker
```

Comment:
By the way, it seems to be been changed to https://gnome.pages.gitlab.gnome.org/tracker/

Reply:
Changed